test/parallel_api/iterator/zip_iterator.h

```diff
@@ -0,0 +1,50 @@
+// -*- C++ -*-
+//===-- zip_iterator.h ---------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ZIP_ITERATOR_H
```

Comment:
I suggest renaming this to zip_iterator_funcs.h to better reflect what is here.

Reply:
Done